dialogTemplate/fr.json

```diff
@@ -0,0 +1,210 @@
+	"slotTypes": [
+		{
+			"name": "MiniGame",
+			"matchingStrictness": null,
+			"automaticallyExtensible": false,
+			"useSynonyms": true,
+			"values": [
+				{
+					"value": "FlipACoin",
+					"synonyms": [
+						"pile ou face",
+						"tirer à pile ou face",
+						"tire à pile ou face",
+						"lancer une pièce",
+						"lance une pièce"
+					]
+				},
+				{
+					"value": "RollADice",
+					"synonyms": [
+						"lancer un dé",
+						"lance un dé",
+						"lancer les dés",
+						"lance les dés",
+						"lancer des dés",
+						"lance des dés",
+						"jeter un dé",
+						"jette un dé",
+						"jeter les dés",
+						"jette les dés",
+						"jeter des dés",
+						"jette des dés"
+					]
+				},
+				{
+					"value": "RockPaperScissors",
+					"synonyms": [
+						"pierre papier ciseaux",
+						"pierre feuille ciseaux",
+						"caillou papier ciseaux",
+						"caillou feuille ciseaux",
+						"chifoumi",
+						"chifumi",
+						"shifoumi",
+						"shifumi"
+					]
+				},
+				{
+					"value": "GuessTheNumber",
+					"synonyms": [
+						"nombre mystère",
+						"le nombre mystère",
+						"deviner le nombre mystère",
+						"devine le nombre mystère",
+						"deviner un nombre mystère",
+						"devine un nombre mystère",
+						"deviner le nombre",
+						"devine le nombre",
+						"deviner un nombre",
+						"devine un nombre"
+					]
+				}
+			]
+		},
+		{
+			"name": "HeadsOrTails",
+			"matchingStrictness": null,
+			"automaticallyExtensible": true,
+			"useSynonyms": true,
+			"values": [
+				{
+					"value": "heads",
+					"synonyms": [
+						"face"
+					]
+				},
+				{
+					"value": "tails",
+					"synonyms": [
+						"pile"
+					]
+				}
+			]
+		},
+		{
+			"name": "RockPaperOrScissors",
+			"matchingStrictness": null,
+			"automaticallyExtensible": true,
+			"useSynonyms": true,
+			"values": [
+				{
+					"value": "rock",
+					"synonyms": [
+						"pierre",
+						"caillou"
+					]
+				},
+				{
+					"value": "paper",
+					"synonyms": [
+						"papier",
+						"feuille"
+					]
+				},
+				{
+					"value": "scissors",
+					"synonyms": [
+						"ciseaux",
+						"ciseau"
+					]
+				}
+			]
+		}
+	],
+	"intents": [
+		{
+			"name": "PlayGame",
+			"description": "L'utilisateur demande à Jouer à un mini-jeu",
+			"enabledByDefault": true,
+			"utterances": [
+				"{flip a coin:=>WhichGame}",
+				"s'il te plaît {flip a coin:=>WhichGame}",
+				"{flip a coin:=>WhichGame} s'il te plaît",
+				"faire un jeu",
+				"faisons un jeu",
+				"jouer à {rock paper scissors:=>WhichGame}",
+				"je veux jouer avec toi",
+				"je voudrais jouer avec toi",
+				"jouons ensemble",
+				"joue avec moi",
+				"joue avec moi s'il te plaît",
+				"peux-tu s'il te plaît {flip a coin:=>WhichGame}",
+				"peux-tu {flip a coin:=>WhichGame}",
+				"peux-tu {roll a dice:=>WhichGame}",
+				"jouons à {scissors rock paper:=>WhichGame}",
+				"jouer à {scissors rock paper:=>WhichGame}"
```

Comment:
the slots should be replaced with the french words

Reply:
Ok, but here for example, WhichGame can't change(Slot Name)… True ?
So I must do: "{lancer une pièce:=>WhichGame}" ?
Thanks you.

Reply:
yes the slot names should always stay the same as in english. The same is true for the first slot value in many cases. (the way you did it)